### DISCUSSION TITLE
What is a `0 address check`?

### DISCUSSION THREAD

**Discussion by SquilliamX** [Cyfrin/security-and-auditing-full-course-s23#160] [Q&A] [answered]
what is a `0 address check`, how does someone implement it, and how do we test for it?

**Comment by alymurtazamemon** [comment, marked as answer]
@SquilliamX It is just a validation check to make sure the input address parameter is initialized and is not a `zero address` which is the default value for address type.

You can add the `require` and `assert` statements in the solidity code to make sure the variables of address types are not empty.
